LICENSE

```diff
@@ -1,28 +1,28 @@
-Software License Agreement (BSD License)
+BSD 3-Clause License
+
+Copyright (c) Willow Garage, Inc.
```

Comment:
This should probably be "Open Source Robotics Foundation" - that's what is on the files that have license headers, and is actually a still existent entity...

Reply:
Thanks, I was wondering…